README.rst

```diff
@@ -28,10 +28,12 @@ Installation using the official OpenVPN installers
 Manual installation of OpenVPN GUI
 **********************************
 
-* `Download <https://openvpn.net/index.php/download/community-downloads.html>`_
-  and install OpenVPN
+* First complete installation using an official OpenVPN installer as described above.
```

Comment:
```suggestion
* First install OpenVPN using an official installer as described above.
```
"complete installation" is difficult to parse since it might be "adjective noun" or "verb noun". Simplify the sentence.

Reply:
Thanks for proof-editing my sloppy writing. Updated and force-pushed.